src/storage/mvcc/reader/scanner/backward.rs

```diff
@@ -227,15 +240,39 @@ impl<S: Snapshot> BackwardKvScanner<S> {
         }
         assert!(ts > last_checked_commit_ts);
```

Comment:
What if the data looks like:

```
k1v13, k1v12, k1v11, k1v10
```

And we seek for ts=12, REVERSE_SEEK_BOUND = 3.

1. i=0, `k1v13, k1v12, k1v11, [k1v10]` (`[]` denotes the key that cursor is pointing to)
2. i=1, `k1v13, k1v12, [k1v11], k1v10`
3. i=2, `k1v13, [k1v12], k1v11, k1v10`
4. i=3, break loop.

Now we hit L230 and the function is returned without discovering `k1v13`.

Reply:
Good cache, it's a corner case.

Reply:
Addressed